### PR TITLE
Some fixes

### PR DESCRIPTION
**Pull request by bogem**
Made some fixes according to `gometalinter`

### CODE REVIEW

**Comment by moorereason** [comment]
@bogem,
Thanks for your contribution.  If you're able, please update your commit messages to denote the package.  For example, `commands: Delete unused newUserErrorF` and `parser: Use strings.Contains instead of strings.Index`.

More info in our [commit message guidelines](https://github.com/spf13/hugo/blob/master/CONTRIBUTING.md#git-commit-message-guidelines).

**Comment by bogem** [comment]
Ok, I'll do it tomorrow

**Comment by bogem** [comment]
Done

**Comment by moorereason** [comment]
That's better.  Thanks.

LGTM except, perhaps, removing `newUserErrorF`.  I'm conflicted.  @spf13, what do you think?

**Comment by bogem** [comment]
It's private function and it's never used in projects. I run tests and everything is working

File: commands/hugo.go [hunk 1]
```diff
@@ -77,10 +77,6 @@ func newUserError(a ...interface{}) commandError {
 	return commandError{s: fmt.Sprintln(a...), userError: true}
 }
 
-func newUserErrorF(format string, a ...interface{}) commandError {
```
Comment (bep) [inline commands/hugo.go]: It is not used, but it completes a natural four method API, and it is just a question of time until it is getting used, so don't remove it.

**Comment by bogem** [comment]
Okay, later I will restore that

**Comment by bep** [comment]
@bogem never mind,  I see it has been enough chatter in this PR already. I'll merge.

**Comment by bogem** [comment]
Thanks!

**Pull request closed without merging**